go/vt/mysqlctl/builtinbackupengine.go

```diff
@@ -813,7 +817,7 @@ func (be *BuiltinBackupEngine) backupFile(ctx context.Context, params BackupPara
 	}
 
 	br := newBackupReader(fe.Name, fi.Size(), timedSource)
-	go br.ReportProgress(builtinBackupProgress, params.Logger)
+	go br.ReportProgress(builtinBackupProgress, params.Logger /*restore*/, false)
```

Comment:
The comment should be with the boolean rather than the logger. This is how I've seen it before in the code base:
```
go br.ReportProgress(builtinBackupProgress, params.Logger, false /* restore */)
```

Reply:
Oops :)